### PR TITLE
Add support for cpio

### PR DESCRIPTION
**Pull request by dcfreire**
Got magic bytes from the cpio manual: https://www.mkssoftware.com/docs/man4/cpio.4.asp
And the MIME from: https://en.wikipedia.org/wiki/List_of_archive_formats

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by dcfreire** [comment]
Sure, on it.

File: core.d.ts [hunk 1]
```diff
@@ -147,7 +147,8 @@ export type FileExtension =
 	| 'parquet'
 	| 'class'
 	| 'arj'
-	; // eslint-disable-line semi-style
+	| 'cpio'
+  ; // eslint-disable-line semi-style
```
Comment (sindresorhus) [inline core.d.ts:151]: Incorrect

File: package.json [hunk 2]
```diff
@@ -199,7 +199,8 @@
 		"dwg",
 		"parquet",
 		"class",
-		"arj"
+		"arj",
+    "cpio"
```
Comment (sindresorhus) [inline package.json:203]: Incorrect indentation

**Comment by dcfreire** [comment]
Sorry, I tried fixing the merge conflicts using the github editor and it messed up the formatting, fixing now.

**Comment by sindresorhus** [comment]
Always look over your own diff after changes ;)

**Pull request merged**